### DISCUSSION TITLE
How to process folders?

### DISCUSSION THREAD

**Discussion by gerroon** [freedmand/semantra-python#69] [Q&A]
Hi

The example line seems to scan a couple pdfs. I want to scan folders with subfolders that ahve pdfs. How can I use it?

thanks

**Comment by underwater** [comment]
I have the same question, but I suspect the current setup will only allow you to specifiy a list of documents by name, and will need some tweking to scan folders...

**Reply by underwater** [reply]
i may have rushed with my earlier response, as the docs suggest doing this
`semantra <path_to_speeches>/*.txt`
